config.ts

```diff
@@ -1,6 +1,51 @@
-import fs from 'fs';
-import toml from 'toml';
+import fs from "fs";
+import toml from "toml";
 
-let config = toml.parse(fs.readFileSync('config-instance.toml').toString());
+// Read config from environment variables
```

Comment:
Reading config from environment variable is something i'd like to do but it would require some extra work for myself in the production environment. At this stage i'm deploying to a DigitalOcean droplet (Linux VM) without docker, so this may be easier using `dotenv` `.env` files.